targets/targets.json

```diff
@@ -584,7 +584,9 @@
         "features": ["LWIP", "STORAGE"],
         "release_versions": ["2", "5"],
         "device_name": "MK64FN1M0xxx12",
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "vector_table_size": 1024,
+        "flash_erase_size": 4096
```

Comment:
I asked below, but this is better placement as these 2 new variables are introduced here. What is ``flash_erase_size`` - sector size? Where to get ``vector_table_size`` number from?

Could these 2 be extracted from the arm pack manager (at least flash erase size could be) ?